containers/sign/PwdLabel.tsx

```diff
@@ -21,36 +22,35 @@ export default function PwdInputWithLabel<T extends FieldValues>({
   const type = visible ? 'text' : 'password';
 
   return (
-    <div>
-      <label htmlFor={id}>
+
+    <div className={styles.container}>
+      <label htmlFor={id} className={styles.label}>
         {label}
       </label>
-      <div className='relative'>
+      <div className={styles.inputWrapper}>
```

Comment:
저희 className 명 케밥 케이스로 통일해주기로 컨베션을 정해놨어서,
```suggestion
className={styles['input-wrapper']}
```
으로 일괄 수정 부탁드립니다!